src/app.ts

```diff
@@ -122,4 +184,135 @@ app.get(
   }
 )
 
+app.get(
+  '/joinGame',
+  celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      playerToken: Joi.string().required().min(1).max(250).label('playerToken'),
+      nickname: Joi.string().required().max(20).label('nickname'),
+      gameId: Joi.number().required().min(0).max(999999).label('gameId'),
```

Comment:
I don't think min 0 is right, it should be either min n>0 or non min at all

Reply:
But can't we receive negative number from client?